iroh/src/magicsock/relay_actor.rs

```diff
@@ -623,13 +641,19 @@ impl ActiveRelayActor {
         state: &mut ConnectedRelayState,
         client_stream: &mut iroh_relay::client::ClientStream,
     ) -> Result<()> {
+        const SEND_TIMEOUT: Duration = Duration::from_secs(10); // TODO: what should this be?
```

Comment:
what should this be?

Reply:
contraints are:
- MAX_PACKET_SIZE is set to 64 KiB
- The client sends up to 20 packets at once (if this is too annoying we could take out that batching)
- so worst-case a single send is 200 KiB
- Practically I don't think 64 KiB packets will be sent
- 1500 bytes/packet is more likely
- so ~30 KiB is a more typical max send size

I'm not sure what lower limit of bandwidth we should assume here.

Reply:
set to  `PING_INTERVAL` for now, as discussed